src/MCP2515/MCP2515_Config.h

```diff
@@ -92,7 +92,8 @@ static CanBitRateConfig constexpr BitRate_1000kBPS_12MHz = {0x00, 0x88, 0x01};
 class MCP2515_Config
 {
 public:
-  MCP2515_Config(MCP2515_Io & io);
+  using MicroSecondFunc = std::function<unsigned long()>;
```

Comment:
Instead of copying this function from class to class create a new header `MCP2515_Types.h` and use a typedef. Include this file were needed.